client/src/app/site/motions/components/motion-detail-diff/motion-detail-diff.component.html

```diff
@@ -1,73 +1,77 @@
 <!-- A summary of all changes -->
 <section class="change-recommendation-overview">
-    <strong>
-        {{ 'Summary of changes' | translate }}:
-    </strong>
+    <strong> {{ 'Summary of changes' | translate }}: </strong>
 
     <!--
-    <button os-perms="motions.can_manage" class="btn btn-sm btn-default pull-right"
-        uib-tooltip="{{ 'Note: You have to reject all change recommendations if the plenum does not follow the recommendation. This does not affect amendments.' | translate }}"
-        ng-click="viewChangeRecommendations.rejectAllChangeRecommendations(motion)">
-      <i class="fa fa-thumbs-down"></i>
-      <translate>Reject all change recommendations</translate>
-    </button>
+        <button os-perms="motions.can_manage" class="btn btn-sm btn-default pull-right"
+            uib-tooltip="{{ 'Note: You have to reject all change recommendations if the plenum does not follow the recommendation. This does not affect amendments.' | translate }}"
+            ng-click="viewChangeRecommendations.rejectAllChangeRecommendations(motion)">
+          <i class="fa fa-thumbs-down"></i>
+          <translate>Reject all change recommendations</translate>
+        </button>
     -->
 
     <ul *ngIf="changes.length > 0">
         <li *ngFor="let change of changes">
-          <a href='' (click)="scrollToChangeClicked(change, $event)"
-             [class.amendment]="isAmendment(change)"
-             [class.recommendation]="isChangeRecommendation(change)">
-            <span *ngIf="change.getLineFrom() >= change.getLineTo() - 1" class="line-number">
-                {{ 'Line' | translate }} {{ change.getLineFrom() }}<span *ngIf="isChangeRecommendation(change)"></span>
-            </span>
-            <span *ngIf="change.getLineFrom() < change.getLineTo() - 1" class="line-number">
-                {{ 'Line' | translate }} {{ change.getLineFrom() }} -
-                    {{ change.getLineTo() - 1 }}<span *ngIf="isChangeRecommendation(change)"></span>
-            </span>
-            <span *ngIf="isChangeRecommendation(change)">({{ 'Change recommendation' | translate }})</span>
-            <span *ngIf="isAmendment(change)">@TODO Identifier</span>
-            <span class="operation" *ngIf="isChangeRecommendation(change)">–
-                {{ getRecommendationTypeName(change) }}
-                <!-- @TODO
-                <span ng-if="change.original.getType(motion.getVersion(version).text) == 3">
-                  { change.other_description }
+            <a
+                href=""
+                (click)="scrollToChangeClicked(change, $event)"
+                [class.amendment]="isAmendment(change)"
+                [class.recommendation]="isChangeRecommendation(change)"
+            >
+                <span *ngIf="change.getLineFrom() >= change.getLineTo() - 1" class="line-number">
+                    {{ 'Line' | translate }} {{ change.getLineFrom()
```

Comment:
looks odd. Perhaps try to put that line into a `<span>` or a `<p>`?

try out [here ](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEAeArgGwHwB0qqYCWeBAzgA4CGUABAFRQDmAkgGYC8u4AFjU3AB0AmABkiUOADEAThAC2ACgCUtbB1pg+zISPGSAKhBW0AtLQCM3TZipkyXEMUmmo6eQCM4M7qWDBaAHJ9OEDaAB9aGBkaMlt4WgBfRNp-TW0BYTgxCWk5JVVk-FRKGgZmdkciMgBhDLgAJThIeXkEABMqGCJoRS1+OGVfVAB6UqhSUfHJkeIZrFJ8EAAaEAgKbugyZFAqGTkAdwAFPYRtlCoANwgidpWQDxiwAGtsgGVqMAkmZGj0OFWPBg8kwAHUeER4KUwHA3mdIURLpCAJ7IcD2e4SMjeGBHGJMeRUZBsKiYbGrABWZAAHgAhJ6vGBvKhtELE0nkkBU6lvb6YOAARXQEHg7LJAJA1Bk2JkaP4WD29woMgkMFBtxgPGQFgATAAGVbKiDY0ExCho5VwGWXOD3GRwACO6CI9rxVAJRKQJPFq2x8iIvxk-19fMFwtFXo5EpgVA86vamuQABZVtEqERnEwagpCWioNBbat0NiDLHzt7sckgA)